tests/test_wagon.py

```diff
@@ -299,44 +263,6 @@ def test_install_package_in_missing_venv(self):
         assert 'virtualenv {0} does not exist'.format(non_existing_venv) \
             in str(ex.value)
 
-    @pytest.mark.skipif(not wagon.IS_LINUX, reason='Irrelevant on non-Linux')
-    def test_get_platform_properties(self):
```

Comment:
`wagon._get_os_properties` just does pretty much `return distro.linux_distribution()`, so this test ends up being `assert linux_distribution() == linux_distribution()`, at the cost of having distro required in the test.
I'm not a big fan of tests that check that we wrote the code in exactly the way that we did write the code